packages/core/src/utilities/VoxelManager.ts

```diff
@@ -26,19 +30,33 @@ export default class VoxelManager<T> {
     [Infinity, -Infinity],
   ] as BoundsIJK;
 
-  // Provide direct access to the underlying data, if any
-  public scalarData: PixelDataTypedArray;
   public map: Map<number, T> | RLEVoxelMap<T>;
   public sourceVoxelManager: VoxelManager<T>;
-  public isInObject: (pointIPS, pointIJK) => boolean;
+  public isInObject: (pointLPS, pointIJK) => boolean;
   public readonly dimensions: Point3;
-  public numComps = 1;
+  public numberOfComponents = 1;
+  public getCompleteScalarDataArray?: () => ArrayLike<number>;
```

Comment:
Nice to see that arraylike is getting used, that makes it much easier to understand things.